concepts/streams/about.md

```diff
@@ -0,0 +1,13 @@
+# About Streams
+
+Java Streams provide a functional, declarative approach to processing collections of data.
+
+Instead of writing explicit loops, Streams let you build **pipelines** of operations — transforming data step-by-step in a clean, readable way.
+
```

Comment:
I would also introduce the intermediate and terminal methods to introduce the possiblity of chaining function.
+ I would also introduce the usage of the lambda with only one function to execute which makes more readable